test/subspace-test-client/src/chain_spec.rs

```diff
@@ -138,6 +146,16 @@ fn create_genesis_config(
                 signing_key: get_from_seed::<OperatorPublicKey>("Alice"),
                 minimum_nominator_stake: 100 * SSC,
                 nomination_tax: Percent::from_percent(5),
+                initial_balances: crate::domain_chain_spec::endowed_accounts()
```

Comment:
How about adding a test to check the balance of the endowed accounts to ensure the domain instantiate path works as expected?

Reply:
I mean a test that check all the endowed account have the exact balance in the domain chain state as what we set here, non-blocker.